code/app/src/main/java/com/example/snackoverflow/AddRecipe.java

```diff
@@ -0,0 +1,81 @@
+package com.example.snackoverflow;
```

Comment:
Why is this file a duplicate of [Nagham's pr](https://github.com/CMPUT301F22T22/SnackOverflow/pull/49)

Reply:
ah shit i didn't know the pr would also merge, I had to merge with their branch to make the fragment work